### PR TITLE
Methods for manipulating camera angles

### PR DESCRIPTION
**Pull request by dmiklic**
Exends the Viewer object public API with methods for manipulating camera angles and positions.

### CODE REVIEW

File: viewer/Viewer.cpp [hunk 1]
```diff
@@ -140,6 +140,31 @@ namespace Enki
 		}
 	}
 	
+	void ViewerWidget::setCamera (QPointF pos, double altitude, double yaw, double pitch)
+	{
+		this->pos = pos;
```
Comment (stephanemagnenat) [inline viewer/Viewer.cpp]: Note that the internal pos member has inverted values, it actually encodes (-x,-y), see Viewer.cpp:690 for its use.

File: viewer/Viewer.cpp [hunk 2]
```diff
@@ -140,6 +140,31 @@ namespace Enki
...
+		this->altitude = altitude;
+		this->yaw = yaw;
```
Comment (stephanemagnenat) [inline viewer/Viewer.cpp]: Yaw is also inverted internally, see Viewer.cpp:789

File: viewer/Viewer.cpp [hunk 3]
```diff
@@ -140,6 +140,31 @@ namespace Enki
...
+		this->pitch = pitch;
```
Comment (stephanemagnenat) [inline viewer/Viewer.cpp]: Pitch is also inverted internally, see Viewer.cpp:795

File: viewer/Viewer.cpp (hunk 1 above)
Comment (stephanemagnenat) [inline viewer/Viewer.cpp]: See https://github.com/aseba-community/aseba/blob/e9e04d830e2eea2da85c292b9d01219f3bf91db3/targets/playground/PlaygroundViewer.cpp#L68

Comment (dmiklic) [inline viewer/Viewer.cpp]: @plsm Can you please fix this?
@rmm-fcul I think this will affect the values we need to put into the Playground.cfg file.

**Comment by dmiklic** [comment]
Recent changes to Enki's Viewer API make this pull request obsolete. Closing.

File: viewer/Viewer.cpp (hunk 1 above)
Comment (plsm) [inline viewer/Viewer.cpp]: The setCamera method sets attribute camera.

    void ViewerWidget::setCamera(QPointF pos, double altitude, double yaw, double pitch)
    {
        camera.pos = pos;
        camera.altitude = altitude;
        camera.yaw = yaw;
        camera.pitch = pitch;
    }

**Pull request closed without merging**